software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -95,7 +95,8 @@ public void install() {
         OsDetails osDetails = getMachine().getMachineDetails().getOsDetails();
         List<String> commands = Lists.newLinkedList();
         if (osDetails.isLinux()) {
-            commands.addAll(installLinux(getExpandedInstallDir()));
+//            commands.addAll(installLinux(getExpandedInstallDir()));
```

Comment:
Any reason to keep this line commented out? Can we just delete it?
We like to be strict about no commented out code - it's only allowed if there is a very clear comment that says why the code might be useful.

Actually, it looks like we are no longer calling `installLinux` at all, which means we have lost a lot of the other stuff it did (e.g. installing logrotate libpam0g-dev libssl0.9.8, etc). Why are we no longer doing that?